minio/datatypes.py

```diff
@@ -214,11 +214,14 @@ def fromxml(cls, element, bucket_name, is_delete_marker=False,
             else (findtext(tag, "ID"), findtext(tag, "DisplayName"))
         )
 
-        tag = find(element, "UserMetadata") or []
         metadata = {}
-        for child in tag:
-            key = child.tag.split("}")[1] if "}" in child.tag else child.tag
-            metadata[key] = child.text
+        selement = find(element, "UserMetadata")
+        if selement:
+            for child in findall(selement, "Items"):
+                key = find(child, "Key")
+                value = find(child, "Value")
+                if key and value:
+                    metadata[key.text] = value.text
```

Comment:
You could directly use `metadata[findtext(child, "Key", True)] =  findtext(child, "Value", True)`

Reply:
let me do that @balamurugana